src/gameAPI.js

```diff
@@ -0,0 +1,60 @@
+import myScoreBoard from './scoreBoard';
+
+const requestURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/';
+
+const scoresBoard = () => {
+  fetch(requestURL, {
+    mode: 'cors',
+    method: 'POST',
+    body: JSON.stringify({
+      name: 'Cricket',
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  })
+    .then((response) => response.json());
+};
```

Comment:
- [ ] Any function that returns promise should be an asynchronous function. Please refactor this function to make it asynchronous like the rest of your functions.